tests/test_unit_schemas.py

```diff
@@ -341,13 +341,27 @@ def test_user_create():
     u = UserCreate(email="a@b.c", password="asd")
     debug(u)
     assert u.slurm_user is None
+    assert u.slurm_accounts == []
     # With valid slurm_user attribute
     u = UserCreate(email="a@b.c", password="asd", slurm_user="slurm_user")
-    debug(u)
     assert u.slurm_user
     # With invalid slurm_user attribute
     with pytest.raises(ValidationError):
-        u = UserCreate(email="a@b.c", password="asd", slurm_user="  ")
+        UserCreate(email="a@b.c", password="asd", slurm_user="  ")
+    # With valid slurm_accounts attribute
+    u = UserCreate(email="a@b.c", password="asd", slurm_accounts=["a", "b"])
+    assert u.slurm_accounts == ["a", "b"]
+    u = UserCreate(
+        email="a@b.c", password="asd", slurm_accounts=[1, "a", True]
+    )
+    assert u.slurm_accounts == ["1", "a", "True"]
```

Comment:
I'd rather not have this coerce-to-string feature in-place, but Pydantic v1 doesn't offer a global configuration for strict mode (ref https://docs.pydantic.dev/1.10/usage/validation_decorator/#coercion-and-strictness). However I think that's what `StrictStr` does, right?

```python
from pydantic import BaseModel
from pydantic import StrictStr


class Model(BaseModel):
    x: str


class StrictModel(BaseModel):
    x: StrictStr


print(Model(x=1))
# x='1'

print(StrictModel(x=1))
# Traceback (most recent call last):
#   File "/tmp/x.py", line 14, in <module>
#     print(StrictModel(x=1))
#   File "pydantic/main.py", line 341, in pydantic.main.BaseModel.__init__
# pydantic.error_wrappers.ValidationError: 1 validation error for StrictModel
# x
#   str type expected (type=type_error.str)
```


(ref #1066)